ios/objective-c/AmpliObjectiveCSampleApp/AmpliObjectiveCSampleApp/Ampli/Ampli.h

```diff
@@ -0,0 +1,760 @@
++ (instancetype _Nonnull) builderBlock:(void (^_Nonnull)(EventOptionsBuilder*_Nonnull))buildBlock;
+@end
+
+@interface EventOptionsBuilder: NSObject
+@property (nonatomic, strong) NSString * _Nullable deviceId;
+@property (nonatomic, strong) NSString * _Nullable userId;
+- (instancetype _Nonnull)initWithOptions:(EventOptions *_Nullable)options;
+- (EventOptions *_Nonnull)build;
+@end
+
+
+@interface Ampli: NSObject
+@property (nonatomic, strong, readonly) Amplitude * _Nullable amplitude;
+@property (nonatomic, assign, readwrite) BOOL disabled;
++ (instancetype _Nonnull)instance;
+- (instancetype _Nonnull )init;
+- (void)load;
+- (void)load:(LoadOptions *_Nullable)options;
+- (void)track:(Event *_Nonnull)event;
+- (void)track:(Event *_Nonnull)event extra:(MiddlewareExtra *_Nullable)extra;
+- (void)track:(Event *_Nonnull)event options:(EventOptions *_Nullable)options extra:(MiddlewareExtra *_Nullable)extra;
+- (void)identify:(NSString *_Nullable)userId properties:(IdentifyProperties *_Nullable)properties;
+- (void)identify:(NSString *_Nullable)userId properties:(IdentifyProperties *_Nullable)properties options:(EventOptions *_Nullable)options;
+- (void)identify:(NSString *_Nullable)userId properties:(IdentifyProperties *_Nullable)properties extra:(MiddlewareExtra *_Nullable)extra;
+- (void)identify:(NSString *_Nullable)userId properties:(IdentifyProperties *_Nullable)properties options:(EventOptions *_Nullable)options extra:(MiddlewareExtra *_Nullable)extra;
+- (void)flush;
+/**
+EventMaxIntForTest
+
+[View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/0.0.0/EventMaxIntForTest)
+
+Event to test schema validation
+
+Owner: Test codegen
+
+- Parameter properties The event's properties
+- Parameter extra Extra untyped parameters for use in middleware.
+*/
+- (void)eventMaxIntForTest:(EventMaxIntForTestProperties *_Nonnull)properties extra:(MiddlewareExtra *_Nullable)extra;
+
+/**
+EventMaxIntForTest
+
+[View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/0.0.0/EventMaxIntForTest)
+
+Event to test schema validation
+
+Owner: Test codegen
+
+- Parameter properties The event's properties
+- Parameter extra Extra untyped parameters for use in middleware.
+*/
+- (void)eventMaxIntForTest:(EventMaxIntForTestProperties *_Nonnull)properties options:(EventOptions *_Nullable)options extra:(MiddlewareExtra *_Nullable)extra;
+
+/**
+EventMaxIntForTest
+
+[View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/0.0.0/EventMaxIntForTest)
+
+Event to test schema validation
+
+Owner: Test codegen
+
+- Parameter properties The event's properties
+*/
+- (void)eventMaxIntForTest:(EventMaxIntForTestProperties *_Nonnull)properties;
+
+/**
+Event No Properties
+
+[View in Tracking Plan](https://data.amplitude.com/test-codegen/Test%20Codegen/events/main/0.0.0/Event%20No%20Properties)
```

Comment:
These links aren't clickable for me in QuickHelp even though [the format looks correct](https://stackoverflow.com/questions/43902959/hyperlink-in-an-xcode-comment). Any ideas?

![image](https://user-images.githubusercontent.com/5790872/145948713-9622fd68-9c18-48f3-b06f-6028c8c16608.png)

Reply:
I believe the markup works for swift..However that is not same for objective-c. I have been searching and trying for hour for it..the anchor tag seems better but it is not clickable in quickhelp... Do you know if there is a way for this link in objective-c?

Reply:
Unfortunately no. I also tried a bunch of things that didnt work :(